app/views/nominations/request_nomination_invite/success.html.erb

```diff
@@ -3,19 +3,21 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
 
-    <%= govuk_panel title_text: "Your school has been sent a link", text: nil, classes: "govuk-!-margin-bottom-7" %>
+  <h1 class="govuk-heading-xl">Check your school email</h1>
```

Comment:
One minor thing - could you also set the `<title>` tag to the same text as this h1 please? (on line 1)